mlos_bench/mlos_bench/environments/local/local_env.py

```diff
@@ -215,6 +216,37 @@ def _normalize_columns(data: pandas.DataFrame) -> pandas.DataFrame:
             data.rename(str.rstrip, axis='columns', inplace=True)
         return data
 
+    # All timestamps in the telemetry data must be greater than this date
+    # (a very rough approximation for the start of this feature).
+    _MIN_TS = datetime(2024, 1, 1, 0, 0, 0, tzinfo=pytz.UTC)
+
+    @staticmethod
+    def _datetime_parser(datetime_col: pandas.Series) -> pandas.Series:
```

Comment:
Could potentially move this and `_normalize_columns` to `mlos_bench.util` in the future.